src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -5099,8 +5131,10 @@ namespace ZoneTempPredictorCorrector {
                 SNLoad = 0.0;
             }
 
-            // Hybrid modeling start: Added by Sang Hoon Lee May 2015
-            if ((HybridModelZone(ZoneNum).InfiltrationCalc || HybridModelZone(ZoneNum).InternalThermalMassCalc) && (!WarmupFlag) && (!DoingSizing)) {
+            // Hybrid modeling start: Added by Sang Hoon Lee May 2015, updated by Han Li 2018
+            if ((HybridModelZone(ZoneNum).InfiltrationCalc_T || HybridModelZone(ZoneNum).InternalThermalMassCalc_T ||
+                 HybridModelZone(ZoneNum).PeopelCountCalc_T) &&
+                (!WarmupFlag) && (!DoingSizing)) {
```

Comment:
It would be cleaner to move the entire hybrid model section to a separate function. Is it using very many local variables from `CorrectZoneAirTemp`? Instead of this complex if, this entire block (or the new call to a separate function) should be wrapped with `if HybridModel::FlagHybridModel`. No need to evaluate the rest if this is false. Similar comments for predictor, and for contaminant predictor and corrector.